extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -7,12 +7,17 @@ import * as vscode from 'vscode';
 import * as path from 'path';
 import { promises as fs } from 'fs';
 import * as utils from '../common/utils';
-import { errorFindingBuildFilesLocation } from '../common/constants';
-import * as mssql from 'mssql';
-import * as vscodeMssql from 'vscode-mssql';
 import * as sqldbproj from 'sqldbproj';
+import * as extractZip from 'extract-zip';
+import * as constants from '../common/constants';
+import { HttpClient } from '../common/httpClient';
 
 const buildDirectory = 'BuildDirectory';
+const sdkName = 'Microsoft.Build.Sql';
+const microsoftBuildSqlVersion = '0.1.3-preview';
```

Comment:
may be better to have the version number in a config file or package.json since it will likely be bumped often and is therefore good to isolate from the source code.

Reply:
And allows people to easily modify it in case they want to use a different version (helpful to have customers test fixes without needing a whole release)

Reply:
added a TODO comment to add this in a separate PR